google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/BigtableTableAdminSettings.java

```diff
@@ -89,14 +92,53 @@ public static Builder newBuilder() {
     return new Builder();
   }
 
-  /** Create a new builder preconfigured to connect to the Bigtable emulator. */
+  /**
+   * Create a new builder preconfigured to connect to Bigtable emulator when hostname and port
+   * number are provided through BIGTABLE_EMULATOR_HOST environment variable.
+   */
+  public static Builder newBuilderForEmulator() {
```

Comment:
To keep consistent with the emulator docs, I think this logic needs to be on by default so I think it needs to be implemented in newBuilder():

Something along the lines of:

```java
public static Builder newBuilder() {
  String emulatorEndpoint = System.getenv(BIGTABLE_EMULATOR_HOST);
  if (emulatorEndpoint != null) {
    new Builder().configureForEmulator(emulatorEndpoint);
  }
  return new Builder();
}
public static newBuilderForEmulator(int port) {
  return newBuilderForEmulator("localhost", port);
}
public static newBuilderForEmulator(String hostname, int port) {
  return new Builder().configureForEmulator(hostname, port);
}

private Builder configureForEmulator(String endpoint) {
   // parse hostname:port
   return configureForEmulator(host, port);
}

private Builder configureForEmulator(host, port) {
   ...
}

```